lib/galaxy/webapps/galaxy/controllers/dataset.py

```diff
@@ -215,7 +215,7 @@ def get_edit(self, trans, dataset_id=None, **kwd):
                                 "name": name,
                                 "label": spec.desc,
                                 "options": attributes.options,
-                                "value": attributes.value if attributes.multiple else [attributes.value],
```

Comment:
Wow, that if statement looks to be the opposite of what was intended ? i.e cast to array if attributes.multiple is false ... good catch.